### PR TITLE
Send error report link in v5 email

### PR DESCRIPTION
**Pull request by tie-rack**
The dashboard can't actually display anything about 5.1 feeds yet, but it can produce an error report for processed feeds, so include that in the email for v5 feeds.

This is temporary until we can display the results in the dashboard.

### CODE REVIEW

**Comment by tie-rack** [comment]
This has been deployed to staging for demonstration purposes.

File: notifications/content.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ module.exports = {
            "<p>If you have any questions, please contact <a href='mailto:vip@democracy.works'>vip@democracy.works</a>.</p>" +
            "<p>Thank you!</p>";
   },
+  v5processedFeed: function(message, recipient, group) {
+    return "<p>" + recipient.givenName + ",</p>" +
+           "<p>The data you provided for " + group.description + "'s election has been processed.</p>" +
+           "<p>Please click the link below for an error report.</p>" +
+           "<p><a href='https://" + baseUrl + "/db/feeds/" + message[":public-id"] + "/xml/errors/report'>Go to the Data Dashboard</a></p>" +
```
Comment (sduckett) [inline notifications/content.js]: `message[":public-id"]` because EDN -> JSON?

Comment (tie-rack) [inline notifications/content.js]: Exactly

**Comment by sduckett** [comment]
LGTM

**Pull request merged**